Gruntfile.js

```diff
@@ -733,63 +733,63 @@ module.exports = function (grunt) {
         files: {
           '<%= yeoman.tmp %>/app/components/translations/Messages_cs_CZ.json': [
              '<%= yeoman.tmp %>/components/**/Messages_cs_CZ.json',
```

Comment:
Shouldn't this be changed too ?

Reply:
Its already in a place where there's only our files.

Reply:
Its already in a place where there's only our files (tmp/components).